src/pystack/_pystack/unwinder.cpp

```diff
@@ -74,6 +74,48 @@ DIENameFromScope(Dwarf_Die* die)
     return name == nullptr ? std::nullopt : std::optional<std::string>(name);
 }
 
+static bool
+breakMuslCloneLoop(Dwfl_Frame* state, std::vector<Frame>& frames)
```

Comment:
Let's add some debug logging of some kind when this happens so we can detect what's going on if we end having reports around this

Reply:
Good idea, done.